nixos/modules/services/x11/display-managers/ly.nix

```
@@ -0,0 +1,149 @@
+{ config, lib, pkgs, ... }:
+
+let
+  xcfg = config.services.xserver;
+  dmcfg = xcfg.displayManager;
+  cfg = dmcfg.ly;
+  xEnv = config.systemd.services.display-manager.environment;
```

```suggestion
  xcfg = config.services.xserver;
  dmcfg = config.services.xserver.displayManager;
  cfg = config.services.displayManager.ly;
  xEnv = config.systemd.services.display-manager.environment;
```
eg this. Also this makes it easier to rebase against the displayManager changes done in on of my sddm PRs.